docs/modules/ups_userguide/pages/server-installation.adoc

```diff
@@ -9,6 +9,7 @@
 
 Getting the UnifiedPush Server up and running with an on-premise application server is a four-step setup process:
```

Comment:
Maybe let's remove `on-premise application server` because the instruction now also have a 'serverless' quick start using the docker images.